docs/blog/articles/2021-10-11-KServe-0.7-release.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "The KServe 0.7 release ensures a smooth transition from KFserving to KServe"
+date: 2020-10-11
+description: "KServe 0.7 Release Blog Post"
+type: "blog"
+---
+
+### Authors
+ **Dan Sun**,**Animesh Singh**,**Vedant Padwal** on behalf of the KServe Community
+
+### **What's Changed?**
+
+- #### [Migration job for existing KFServing installation](https://kserve.github.io/website/admin/migration)
+    This helps to migrate existing inference services running in the cluster from KFServing to KServe without downtime. 
+    The script installs `KServe` from 0.7 release and deletes your `KFServing` installation after migrating the inference services from `serving.kubeflow.org` to `serving.kserve.io`.
+<br>
+- `InferenceService` API group is changed from `serving.kubeflow.org` to `serving.kserve.io` [#1826](https://github.com/kserve/kserve/issues/1826).
+- Python SDK name is changed from `kfserving` to `kserve` [#1827](https://github.com/kserve/kserve/issues/1827).
+- KServe Installation manifests [#1824](https://github.com/kserve/kserve/issues/1824).
+- Model-web-app is separated out of the kserve repository to [#1820](https://github.com/kserve/kserve/issues/1820).
+- Docs and examples are moved to separate repository https://github.com/kserve/website.
+- KServe images are migrated to kserve docker hub account.
+- v1alpha2 API group is deprecated [#1850](https://github.com/kserve/kserve/issues/1850).
+
+### **🌈 What's New?**
+
+- #### ModelMesh project is joining KServe under repository [modelmesh-serving](https://github.com/kserve/modelmesh-serving)!
+    ModelMesh is designed for high-scale, high-density and frequently-changing model use cases.
+    ModelMesh intelligently loads and unloads AI models to and from memory to strike an intelligent 
+    trade-off between responsiveness to users and computational footprint.
+        To learn more about ModelMesh features and components, check out the ModelMesh announcement blog. [ModelMesh release blog](https://developer.ibm.com/blogs/kserve-and-watson-modelmesh-extreme-scale-model-inferencing-for-trusted-ai/)
+<br>  
+- #### (Alpha feature) Raw kubernetes deployment support,Istio/Knative dependency is now optional [#1508](https://github.com/kserve/kserve/issues/1508),[#1830](https://github.com/kserve/kserve/issues/1830).
+    KFServing totally depends on Knative to bring serving features like scale to zero, Knative Pod Autoscaler(KPA)
+    and canary rollouts to ML/DL deployments, and the serving runtime contract requires HTTP as the inboundtransport.
+    There are use cases where Knative would be a great solution but where HTTP is not used. Especially, due to safety 
+    concerns, some cases need alternative transports like UDP or TCP or even custom protocols. So, we need an effective 
+    scenario to solve the problem, make Knative Serving and native serving should be both available for the Kubernetes cluster.
```

Comment:
This is not quite accurate which needs to be rephrased. The raw kubernetes deployment is for lightweight dependency and unlocks the Knative limitations.